src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -221,34 +221,77 @@ export interface TransitionDirectionalAnimations {
 }
 ```
 
-The following example shows all the necessary properties to define a custom `fade` animation:
+The following example shows all the necessary properties to define a custom transition animation.
+
+In this instance, the custom animation `crash` is defined inside a `<style is:global>` tag in your root layout file:
```

Comment:
"crash" sounds so destructive and somehow like "broken". Surely there's a more cheerful name?